### ISSUE TITLE
Stochastic Gradient Descent for linear regression

### ISSUE THREAD

**Issue by priyagunjate** [prodevans/LEAP2.0#15]
To implement stochastic gradient descent to optimize a linear regression algorithm on Boston House Prices dataset which is already exists in sklearn as a sklearn.linear_model.SGDRegressor.here,SGD algorithm is defined manually and then comapring the both results.Linear regression is technique to predict on real values.
## Objective:

 To Implement stochastic gradient descent on Bostan House Prices dataset for linear Regression 
 
 * Implement SGD and deploy on Bostan House Prices dataset.
 * Comapare the Results with sklearn.linear_model.SGDRegressor
* Compare the results when Learning rate is constant

**Comment by priyagunjate** [comment]
[SGD.pdf](https://github.com/prodevans/LEAP2.0/files/4667875/SGD.pdf)
